README.md

```diff
@@ -1,24 +1,137 @@
-# README
+# Hello World
+<a name="readme-top"></a>
+<div align="center">
+  <img src="https://github.com/microverseinc/readme-template/raw/master/murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Kindly remove the Microverse logo (replace it with the app logo if you have one)